saleor/discount/migrations/0078_auto_20240328_2015.py

```diff
@@ -0,0 +1,96 @@
+    update_discount_type_checkout_line_task,
+    update_discount_type_order_line_task,
+)
+
+
+def rename_discount_type_promotion_to_catalogue_promotion(_apps, _schema_editor):
+    def on_migrations_complete(**_kwargs):
+        update_discount_type_checkout_line_task.delay()
+        update_discount_type_order_line_task.delay()
+
+    sender = registry.get_app_config("discount")
+    post_migrate.connect(on_migrations_complete, weak=False, sender=sender)
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("discount", "0077_merge_20240307_1217"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="checkoutdiscount",
+            name="type",
+            field=models.CharField(
+                choices=[
+                    ("sale", "Sale"),
+                    ("voucher", "Voucher"),
+                    ("manual", "Manual"),
+                    ("promotion", "Promotion"),
+                    ("order_promotion", "Order promotion"),
+                    ("catalogue_promotion", "Catalogue promotion"),
+                ],
+                default="manual",
+                max_length=64,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="checkoutlinediscount",
+            name="type",
+            field=models.CharField(
+                choices=[
+                    ("sale", "Sale"),
+                    ("voucher", "Voucher"),
+                    ("manual", "Manual"),
+                    ("promotion", "Promotion"),
+                    ("order_promotion", "Order promotion"),
+                    ("catalogue_promotion", "Catalogue promotion"),
+                ],
+                default="manual",
+                max_length=64,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="orderdiscount",
+            name="type",
+            field=models.CharField(
+                choices=[
+                    ("sale", "Sale"),
+                    ("voucher", "Voucher"),
+                    ("manual", "Manual"),
+                    ("promotion", "Promotion"),
+                    ("order_promotion", "Order promotion"),
+                    ("catalogue_promotion", "Catalogue promotion"),
+                ],
+                default="manual",
+                max_length=64,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="orderlinediscount",
+            name="type",
+            field=models.CharField(
+                choices=[
+                    ("sale", "Sale"),
+                    ("voucher", "Voucher"),
+                    ("manual", "Manual"),
+                    ("promotion", "Promotion"),
+                    ("order_promotion", "Order promotion"),
+                    ("catalogue_promotion", "Catalogue promotion"),
+                ],
+                default="manual",
+                max_length=64,
+            ),
+        ),
+        migrations.RunPython(
```

Comment:
Please split that migration into two different ones:
- one with the alters
- one that will run data update